### PR TITLE
Add UIBlockType DownloadButton [RN35869] [DCP204668]

### PR DESCRIPTION
**Pull request by HansSolos**
Doucmentation of new UIBlockType 'DownloadButton '.

### CODE REVIEW

**Review by HansSolos — COMMENTED** [review]
Update relative path examples.

File: src/DataMiner/SLNetTypes/Automation/AutomationDownloadButtonOptions.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        /// </summary>
+        /// <value><c>true</c> to immediately start the download without clicking the button; otherwise, <c>false</c>.</value>
+        /// <remarks>
+        /// <para>Default: <c>false</c>.</para>
+        /// </remarks>
+        public bool StartDownloadImmediately { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating which file must be downloaded. This URL can be either absolute path or relative path.
+        /// When using an absolute URL, then it is recommended that the file is public accesible from the web.
```
Comment (lander-vanhaverbeke-skyline) [inline src/DataMiner/SLNetTypes/Automation/AutomationDownloadButtonOptions.cs:45]: I would remove the lines on line nrs 45 and 46. I think they only add confusion.

**Review by lander-vanhaverbeke-skyline — CHANGES_REQUESTED** [review]
I suggest to change the wording a bit about the absolute/relative URL's.

**Review by lander-vanhaverbeke-skyline — APPROVED** [review]

**Review by lander-vanhaverbeke-skyline — APPROVED** [review]

**Review by HansSolos — COMMENTED** [review]
Remove whitespace

**Review by lander-vanhaverbeke-skyline — APPROVED** [review]

**Review by HansSolos — COMMENTED** [review]
OK for me.
Asked Lander for optional look/opinion

**Review by lander-vanhaverbeke-skyline — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**